### DISCUSSION TITLE
Missing Data Grid Component in Figma

### DISCUSSION THREAD

**Discussion by Mister-Miso** [microsoft/fluentui#33211] [Q&A] [answered]
Will the Figma library for Fluent v9 be updated with a data grid component soon?
Link to Figma Page: https://www.figma.com/design/IkFEUiNq2mN4yjsA6QbrOb/Microsoft-Fluent-2-Web-(Community)?node-id=1-840&t=dewD1mlFUHg1uB8j-1

**Comment by ValentinaKozlova** [comment]
Hello @Mister-Miso, thanks to paying attention to it. I wrote to the design team

**Reply by ValentinaKozlova** [reply, marked as answer]
Hey @Mister-Miso,
I got an answer from the design team. Unfortunately, updating external kit is not the highest priority now. And it's unclear when it'll be updated.
